pytests/test_operators.py

```diff
@@ -196,6 +196,41 @@ def count(count, event_count):
     )
 
 
+def test_fold_epoch():
+    def extract_id(event):
+        return (event["user"], event)
+
+    def builder(event):
+        return {event["type"]: 1}
```

Comment:
Ok, will fix both the code and the test

Reply:
Err, sorry, I misspoke here. I meant `{}`, since this builder should just take the `key` and not the `value` (since the "empty" accumulator wouldn't be able to know about any values yet). The folder will have to add the event type because that's the first point with values.

Or if you want to get fancy, you could return `defaultdict(lambda: 0)` to streamline the folder code.

Reply:
That makes sense, it wasn't clear to me what was the advantage of taking the whole event in the builder, given we call the folder right after, but I assumed it could be useful in some cases. This makes the rust side nicer too, so I'm going to update the code, and then the description of the PR too.

> Or if you want to get fancy, you could return defaultdict(lambda: 0) to streamline the folder code.

Hey, that's a python stdlib piece I didn't know of, I'm going to use it immediately :)